src/containers/Dashboard.vue

```diff
@@ -15,7 +15,7 @@
           class="dashboard-nav"
           @editProfile="editProfile"
           @toggleRequirementsBar="toggleRequirementsBar"
-          :isBottomPreview="bottomBar.isPreview"
+          :isBottomPreview="false"
```

Comment:
If `isBottomPreview` is always going to be `false`, can we just remove that prop and any of its dependencies from NavBar entirely?

Reply:
removed